routes/profileRoute.js

```diff
@@ -0,0 +1,12 @@
+var router = require("express").Router();
+const { validation } = require("../commonFunction/validationFun");
+const { create_2 } = require("../validators/user");
+
+const profileController = require("../controller/profileController");
+
+router.post("/profile", [create_2(), validation], profileController.profile);
```

Comment:
create_2 this is not proper variable name, function name itself define what that function actually does